k8s/services/socks-proxy-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: socks-proxy-service
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "socks-proxy-static-ip"
+spec:
+  type: LoadBalancer
```

Comment:
still not sure I fully understand this ([here](https://cloud.google.com/kubernetes-engine/docs/concepts/service-load-balancer?hl=fr#load_balancer_types) they mention that a GKE annotation might be necessary) but down to try it 👍

Reply:
Me neither, added "ca mange pas de pain".